pom.xml

```diff
@@ -112,6 +113,53 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>${frontend-maven-plugin.version}</version>
+                <configuration>
+                    <nodeVersion>v5.6.0</nodeVersion>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>install node and npm</id>
+                        <goals>
+                            <goal>install-node-and-npm</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                    </execution>
+                    <execution>
+                        <id>npm configure</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                        <configuration>
+                            <arguments>config set strict-ssl false</arguments>
```

Comment:
I always consider disabling ssl as a hack :)

The error:
`[Thread-2] [ERROR] npm ERR! fetch failed https://registry.npmjs.org/asn1/-/asn1-0.2.3.tgz
[Thread-2] [WARNING] npm WARN retry will retry, error on last attempt: Error: unable to get local issuer certificate`

CI machines are configured to pull only from Artifactory using our Okta NPM registry `registry=https://artifacts.aue1d.saasure.com/artifactory/api/npm/npm-okta-master`. If I recall correctly, the npmrc also includes the path to our certificate through `cafile=FOO`.

But you are not pulling from Artifactory but `https://registry.npmjs.org` which definitely does not match or cafile. Also, you are using a maven plugin so I bet https://registry.npmjs.org is not trusted by our maven installations in CI. Since this is open/public, using Artifactory will not be an option.

So, you my suggestion is:
1. This should be reproducible locally. Once you do, trust the https://registry.npmjs.org cert in the maven local cert store and for your global npm installation. Once you succeed in building this locally, then:
2. Request Eng Productivity to repeat for CI machines.